src/custom/api/coingecko/index.ts

```diff
@@ -99,5 +99,5 @@ export function toPriceInformation(priceRaw: CoinGeckoUsdQuote | null): PriceInf
   }
 
   const { usd } = priceRaw[token]
-  return { amount: usd.toString(), token }
+  return { amount: usd.toString(), token, oppositeAmount: '1' }
```

Comment:
i find oppositeAmount missliading.

If we can really not add from/to, would it be better `amountInputted` or something?